lib/ffi.dart

```diff
@@ -109,6 +109,24 @@ Future<void> ffiRemoveDevice(String deviceId) async {
   return;
 }
 
+Future<String> ffiAuthorizeEmail(String emailAddress) async {
+  final json = await _bindings
+      .approveDevice(emailAddress.toPointerChar())
+      .cast<Utf8>()
+      .toDartString();
+  final result = BaseResponse.create()..mergeFromProto3Json(jsonDecode(json));
```

Comment:
I think we need to check for API error here

Reply:
Oops, I forgot to pass that change last night. Thanks!!